src/js/app.js

```diff
@@ -0,0 +1,95 @@
+var picArray = [];
+var count = 0;
+
+
+$(document).ready(function(){
+
+  $('#textInput').focus();
+
+	$('#formId').on('submit', function(e){
+
+		e.preventDefault();
+
+
```

Comment:
The linter will probably yell at you for this, but make sure to remove any unnecessary newlines. In a professional environment, it actually makes code harder to read.